src/coreclr/inc/corinfo.h

```diff
@@ -3063,7 +3063,8 @@ class ICorDynamicInfo : public ICorStaticInfo
     // return the native entry point to an EE helper (see CorInfoHelpFunc)
     virtual void* getHelperFtn (
```

Comment:
Is the JIT going to use both the code pointer and the method handle when callling this method with non-null `pMethod`?

Reply:
It does not have to be a separate API. It can be one API with two optional out args. Using `ppIndirection` to indicate whether you are interested in the actual entrypoint would look odd. I am thinking something like this:

```
void getHelperFtn(CorInfoHelpFunc ftnNum, 
    CORINFO_CONST_LOOKUP* addr, // optional [OUT]
    CORINFO_METHOD_HANDLE* method) // optional [OUT]
```